vmc/resource_vmc_sddc_test.go

```diff
@@ -156,3 +158,95 @@ resource "vmc_sddc" "sddc_1" {
 		sddcName,
 	)
 }
+
+func TestBuildAwsSddcConfig(t *testing.T) {
+	type test struct {
+		input    map[string]interface{}
+		expected model.AwsSddcConfig
+	}
+
+	tests := []test{
+		{input: map[string]interface{}{
+			"sddc_name":          "testName1",
+			"region":             "us-east-1",
+			"provider_type":      ZeroCloudProviderType,
+			"num_host":           MinHosts,
+			"host_instance_type": HostInstancetypeI3,
+		},
+			expected: model.AwsSddcConfig{
+				Name:             "testName1",
+				Region:           "us-east-1",
+				Provider:         ZeroCloudProviderType,
+				NumHosts:         MinHosts,
+				HostInstanceType: String(model.SddcConfig_HOST_INSTANCE_TYPE_I3_METAL),
+				Size:             String(MediumSDDCSize),
+				DeploymentType:   String(SingleAvailabilityZone),
```

Comment:
Same here, don't repeat the whole struct here both for `input` and for `expected`. You can construct the input and expected structs in your testcase, just put what is different in this test compared to the next tests.